proxy/src/proxy.rs

```diff
@@ -110,6 +110,19 @@ static COMPUTE_CONNECTION_LATENCY: Lazy<HistogramVec> = Lazy::new(|| {
     .unwrap()
 });
 
+static CONTROL_PLANE_LATENCY: Lazy<HistogramVec> = Lazy::new(|| {
+    register_histogram_vec!(
+        "proxy_compute_connection_control_plane_latency_seconds",
+        "Time proxy spent talking to control-plane/console while trying to establish a connection to the compute endpoint",
+        // http/ws/tcp, true/false, true/false, success/failure
+        // 3 * 2 * 2 * 2 = 24 counters
+        &["protocol", "cache_miss", "pool_miss", "outcome"],
```

Comment:
Do you think, we need all this slices? I am not sure about `cache_miss` and `pool_miss`.

Reply:
I intend it to mirror the `COMPUTE_CONNECTION_LATENCY` histogram. I would like to compute proxy's sole compute_connection latency by using an expression of the form `proxy_compute_connection_latency_seconds - proxy_compute_connection_control_plane_latency_seconds`

Reply:
hmm, good point!